src/services/lessons/hooks/setCourseId.js

```diff
@@ -0,0 +1,16 @@
+const { BadRequest } = require('@feathersjs/errors');
+
+const setCourseId = (context) => {
+	const { courseId } = context.params.route || {};
+	if (!courseId) {
```

Comment:
Wird die route überhaupt getriggert, wenn courseId gefordert ist aber nicht angebene wird?

Reply:
Wird benötigt für den (internen) model Services. Bei dem wird der gleiche hook verwendet, aber wenn params nicht richtig gesetzt wird, ist in der route keine courseId vorhanden.

Reply:
sprich der hook wird zwei mal hintereinander verwendet? Sollten wir den internen teil nicht clean halten?